cwf/gateway/configs/sessiond.yml

```diff
@@ -12,12 +12,14 @@
 # limitations under the License.
 
 log_level: INFO
+print_grpc_payload: false
```

Comment:
Maybe magma_prinr_grpc_payload? Will be consistent with pipelined and the env variable name

Reply:
Well the way I was told this was originally:

`magma_print_grpc_payload` was supposed to be the envar 
`print_grpc_payload` was the original setting

All other systems uses `print_grpc_payload` on their yml confs (directoryd, magmad, subscriberdb. etc)

Reply:
Good point maybe I should just fix pipelined